src/models/GatewayAccount.class.js

```diff
@@ -40,7 +41,19 @@ class GatewayAccount {
     this.disableToggle3ds = gatewayAccountData.payment_provider === 'stripe'
     this.externalId = gatewayAccountData.external_id
     this.emailCollectionMode = gatewayAccountData.email_collection_mode
-    this.gatewayAccountCredentials = gatewayAccountData?.gateway_account_credentials?.map(
+    this.emailNotifications = {
+      paymentConfirmed: {
+        enabled: gatewayAccountData.email_notifications?.PAYMENT_CONFIRMED?.enabled ?? false,
+        templateBody: gatewayAccountData.email_notifications?.PAYMENT_CONFIRMED?.template_body ?? null,
+        version: gatewayAccountData.email_notifications?.PAYMENT_CONFIRMED?.version ?? null
```

Comment:
not super important but you probably don't need the nullish coalescing operator here, unless you explicitly want to be `null` instead of `undefined`